dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -203,52 +188,41 @@ await foreach (ChatMessage message in choice.GetMessageStreaming(cancellationTok
     /// </summary>
     /// <param name="instructions">Optional chat instructions for the AI service</param>
     /// <returns>Chat object</returns>
-    private protected static ChatHistory InternalCreateNewChat(string? instructions = null)
+    private protected static OpenAIChatHistory InternalCreateNewChat(string? instructions = null)
     {
         return new OpenAIChatHistory(instructions);
     }
 
-    private protected List<ITextCompletionStreamingResult> InternalGetTextCompletionAsChat(string text, CompleteRequestSettings requestSettings, CancellationToken cancellationToken)
-    {
-        return new List<ITextCompletionStreamingResult>
-        {
-            new ChatCompletionAsTextResult(
-                (cancellationTokenInvoke) => this.InternalCompleteTextUsingChatStreamAsync(text, requestSettings, cancellationTokenInvoke),
-                (cancellationTokenInvoke) => this.InternalCompleteTextUsingChatAsync(text, requestSettings, cancellationTokenInvoke))
-        };
-    }
-
-    /// <summary>
-    /// Creates a completion for the prompt and settings using the chat endpoint
-    /// </summary>
-    /// <param name="text">The prompt to complete.</param>
-    /// <param name="requestSettings">Request settings for the completion API</param>
-    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
-    /// <returns>Text generated by the remote model</returns>
-    private async Task<string> InternalCompleteTextUsingChatAsync(
+    private protected async Task<IReadOnlyList<ITextCompletionResult>> InternalGetTextCompletionAsChatAsync(
```

Comment:
I think this should probably be plural `Completions`, as it returns multiple results, and to align with `GetStreamingChatCompletionsAsync`. The public methods appear plural, but internal ones are singular.